src/planscape/planscape/filters.py

```diff
@@ -0,0 +1,5 @@
+from django_filters import filters
+
+
+class CharArrayFilter(filters.BaseCSVFilter, filters.CharFilter):
+    pass
```

Comment:
Just doublechecking that this is meant to pull in the BaseCSVFilter and Charfilter, but not be otherwise implemented?